File: level 89/homework/hw.js
// მომხმარებლისგან წინადადების მიღება
let sentence = prompt("შეიყვანეთ წინადადება: ");

// წინადადების სიტყვებად გადანაწილება, შემოტრიალება და გაერთიანება
let reversedSentence = sentence.split(' ').reverse().join(' ');
console.log("შემოტრიალებული სიტყვები:", reversedSentence);

File: level 89/homework/hw.py
# მომხმარებლისგან სტრინგის და ორი ინდექსის მიღება
text = input("გთხოვთ, შეიტანოთ სტრინგი: ")
start = int(input("შეიყვანეთ დამწყები ინდექსი: "))
end = int(input("შეიყვანეთ დასრულების ინდექსი: "))

# slicing-ის გამოყენება
sliced_text = text[start:end]
print("სტრინგი შემდეგი ინდექსებიდან:", sliced_text)

# სიტყვების შემოტრიალება
reversed_sentence = ' '.join(text.split()[::-1])
print("შემოტრიალებული სიტყვები:", reversed_sentence)
